SpatialGDKEditorToolbar/SpatialGDKEditorToolbar.uplugin

```diff
@@ -20,11 +20,5 @@
 			"LoadingPhase": "Default",
 			"WhitelistPlatforms": [ "Win64" ]
 		}
-	],
```

Comment:
This needs to be here, otherwise we will get a dependency warning wehn building the plugin

Reply:
When building the plugin in standalone mode, which we do for CI. Will revert.